cmd/root.go

```diff
@@ -66,11 +67,21 @@ func Execute(build string) {
 func init() {
 	cobra.OnInitialize(initConfig)
 
+	viper.SetConfigName(".babylon")
+	viper.SetConfigType("yaml")
+	// viper.AddConfigPath(".")
+
```

Comment:
It's actually being done that way kind of already. The defaults on init add configuration paths for $HOME and the current directory. In the case of absolute call to the model, however, once we have established the original path for the model, we check to see if a config has been loaded, if not we add the original path of the model in as a location and try to read it in again. That way you could make sure the config is committed with the models and it would work whether you call it locally, or absolutely

Reply:
I'll double check the spec though. I need to find a way to enforce priority for viper's configuration path searching